src/test-utils/matchers/index.tsx

```diff
@@ -0,0 +1,74 @@
+import userEvent from "@testing-library/user-event";
+import AuthProvider from "components/AuthProvider/AuthProvider";
+import useAuth from "hooks/useAuth/useAuth";
+import { screen, render } from "@testing-library/react";
+
+const expectToBeOnPage = (
+  container: HTMLElement,
+  myPath: string,
+  expectedPage: string
+) => {
+  const expectedPath = `/${expectedPage}`;
+  expect(myPath).toEqual(expectedPath);
+  expect(container.querySelector("ion-title")).toHaveTextContent(
+    new RegExp(expectedPage, "i")
+  );
+};
+
+const expectToRenderHeaderWithTitleAndLogoutButton = (
```

Comment:
Is this an appropriate abstraction for rendering header correctly across Home, Offers, Donors?